packages/lesswrong/components/comments/CommentsItem/CommentBody.jsx

```diff
@@ -29,6 +33,20 @@ const styles = theme => ({
     },
     '&:hover .read-more-default': {
       display:"none"
+    },
+    '& .read-more-f-tooltip': {
+      display: "none",
+      position: "absolute",
+      background: "rgba(0,0,0,.6)",
+      color: "white",
+      padding: "7px 10px",
+      width:156,
```

Comment:
Is this the empirical width of the text? That seems very brittle; it will break if we change font, or if the user's browser ignores our font-size choice (as some browsers do).